From d635b8655bab1f945cd3bd39a45d576d9709b895 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:06:17 -0400
Subject: [usb] uas: s/response_ui/response_iu/

Message-id: <1410887463-20674-306-git-send-email-dzickus@redhat.com>
Patchwork-id: 95181
O-Subject: [RHEL7 PATCH 305/591] uas: s/response_ui/response_iu/
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit e52e031498cb51aff4f80a19a56700a127cf2a9a
Author: Hans de Goede <hdegoede@redhat.com>
Date:   Wed Oct 23 14:27:09 2013 +0100

    uas: s/response_ui/response_iu/

    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/storage/uas.c b/drivers/usb/storage/uas.c
index 0ee5a05..33f9dcd 100644
--- a/drivers/usb/storage/uas.c
+++ b/drivers/usb/storage/uas.c
@@ -46,7 +46,7 @@ struct uas_dev_info {
  struct usb_anchor sense_urbs;
  struct usb_anchor data_urbs;
  int qdepth, resetting;
- struct response_ui response;
+ struct response_iu response;
  unsigned cmd_pipe, status_pipe, data_in_pipe, data_out_pipe;
  unsigned use_streams:1;
  unsigned uas_sense_old:1;
diff --git a/include/linux/usb/uas.h b/include/linux/usb/uas.h
index 5499ab5..1404178 100644
--- a/include/linux/usb/uas.h
+++ b/include/linux/usb/uas.h
@@ -79,7 +79,7 @@ struct sense_iu {
  __u8 sense[SCSI_SENSE_BUFFERSIZE];
 };
 
-struct response_ui {
+struct response_iu {
  __u8 iu_id;
  __u8 rsvd1;
  __be16 tag;
-- 
1.7.1